README.md

```diff
@@ -51,11 +51,11 @@ In Linux, these can all be acquired at the same time with the following command:
   - Change to the `$MENGE_ROOT/projects/g++` directory
   - type: `make`
   
-It will attempt to build both Menge and its Plug-ins.  It will not attempt to build the
+It will build both Menge and its Plug-ins.  It will not build the
 documentation (see below for instructions).  It does this by calling CMake and then
 immediately make.
 
-It uses whatever default C++ compiler, if one can be found.  It also defaults to building a
+It uses whatever default C++ compiler.  It also defaults to building a
```

Comment:
Both before and after suffer from an erroneous "whatever".  I.e., it should be "It uses the system's default C++ compiler."